src/test/java/ua/hillel/ContactUsTest.java

```diff
@@ -0,0 +1,38 @@
+package ua.hillel;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ua.hillel.pages.ContactUsPage;
+import ua.hillel.pages.MainPage;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ContactUsTest extends BaseTest {
+    @Test
+    public void checkContactUsForm()  {
+
+        Properties userData = new Properties();
+        try {
+            userData.load(new FileReader("src/test/resources/userdata.properties"));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
can be moved to separate method/class, e.g.  [example from lesson](https://github.com/Nikots/mavenHillelDemoTestNg/blob/main/src/test/java/hillel/lesson24_files_part1/ConfigLoader.java)
```suggestion
        Properties userData = new Properties();
        try {
            userData.load(new FileReader("src/test/resources/userdata.properties"));
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
```